api/database/seeders/DatabaseSeeder.php

```diff
@@ -111,6 +111,23 @@ public function run()
         $pool = Pool::whereNotNull('published_at')->inRandomOrder()->first();
         $this->seedPoolCandidate($applicant, $pool);
         $this->seedAwardExperienceForPool($applicant, $digitalTalentPool);
+        $applicantUserSkills = DB::table('user_skills')
```

Comment:
Is there a reason you're not using the ORM to get and update these models?
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/68351e37-82d3-46ea-a741-6cd9905ec400)

Reply:
Facade seemed more straightforward to me in this case. I think this bit of seeding will probably be in flux, so I just rolled with what seemed simpler to me.

Reply:
Nay. Sorry @vd1992 - for consistency, please use Eloquent models everywhere (except for in migrations).

Reply:
As an aside, I found it simpler, now I know why. 
I was originally trying to go `$applicant->userSkills()`, which obviously didn't work and gave up in favour of facade. 
I realized the issue looking at Peter's picture above. 

I just needed to not use the `()` 🤦 

Switched in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7440/commits/2c6b92625783fd3d2095d626d9ae06d2f18673c7

Reply:
Looks great!  BTW, the screenshot is from [Tinker](https://laravel.com/docs/10.x/artisan#tinker), which I've started using more lately.

Reply:
Much to dig into 🤔 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/911f7f65-1972-4d8c-ac4a-c0954810ea16)